### PR TITLE
Detection loop in linked list

### PR DESCRIPTION
**Pull request by NupurHardiya**
Issue no. #1403 
--------------------------------------------------------------------------------------------------------------------------------------------
Here are the steps involved in the approach used to detect of a cycle in a linked list using a hash table:

1. Create a hash table to store the addresses of all the nodes in the linked list. Start traversing the linked list, and for each node, 2. check if the node's address is already in the hash table.
3. If the node's address is already in the hash table, then the linked list contains a loop.
4. Otherwise, the node's address is added to the hash table, and the traversal continues.
5. The code returns "The linked list contains a loop" if the linked list contains a loop, and "The linked list does not contain a loop." otherwise.
--------------------------------------------------------------------------------------------------------------------------------------------
- [✅] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.
-------------------------------------------------------------------------------------------------------------------------------------------
detection of loop in LL

### CODE REVIEW

**Review by zugzwang03 — COMMENTED** [review]
name the files in .extension format and remove space over there

**Review by 072arushi — APPROVED** [review]

**Comment by NupurHardiya** [comment]
Hey @zugzwang03 I have done all the changes. please check.

**Review by zugzwang03 — APPROVED** [review]

**Review by Arun9739 — APPROVED** [review]

File: Java/Linked-List/detectionLoopInLinkedList/Node.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Node {
```
Comment (Kumar-laxmi) [inline Java/Linked-List/detectionLoopInLinkedList/Node.java:1]: Put this class in ```Java/Linked-List/detectionLoopInLinkedList/LoopDetection.java```

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Give the name as ```Detect_Loop_In_Linked_List``` to all the files

File: Java/Linked-List/detectionLoopInLinkedList/loopdetection.java/node.java
Comment (zugzwang03) [inline Java/Linked-List/detectionLoopInLinkedList/loopdetection.java/node.java:1]: you placed file inside file, place it in folder rather

**Pull request closed without merging**